go.mod

```diff
@@ -62,8 +62,8 @@ replace (
 
 require (
 	github.com/antihax/optional v1.0.0
-	github.com/rancher/rancher/pkg/apis v0.0.0-20240618122559-b9ec494d4f6f
-	github.com/rancher/shepherd v0.0.0-20240628185912-bd372ac2e2e1
+	github.com/rancher/rancher/pkg/apis v0.0.0-20240701102734-a84c3a2f36c1
```

Comment:
is this expected?

Reply:
is there now no go.mod change?

Reply:
Yes, there no go.mod change now, because of the rebase the commit bellow already changed the go.mod https://github.com/lscalabrini01/rancher/commit/9ada4ef668fb68db7a7819c41667ce8bf74bb2d5